File: Sc.Models/Dtos/ReceivedVouchers/ReceivedVoucherHistoryDto.cs
using Sc.Models.Dtos.ReceivedVouchers.Base;

namespace Sc.Models.Dtos.ReceivedVouchers
{
    public class ReceivedVoucherHistoryDto : BaseReceivedVoucherDto
    {
        public int ReceivedVoucherId { get; set; }
        public ReceivedVoucherHistoryFileDto File { get; set; }
    }
}

File: WebApplication/src/received-vouchers/dtos/received-voucher-history.dto.ts
import { BaseReceivedVoucherDto } from "./base/base-received-voucher.dto";
import { ReceivedVoucherHistoryFileDto } from "./received-voucher-history-file.dto";

export class ReceivedVoucherHistoryDto extends BaseReceivedVoucherDto {
    receivedVoucherId: number;
    file: ReceivedVoucherHistoryFileDto;
}